src/test/java/uk/nhs/hee/tis/trainee/sync/service/PersonServiceTest.java

```diff
@@ -37,7 +37,7 @@
 
 class PersonServiceTest {
 
-  private PersonRepository personRepository;
+  private static PersonRepository personRepository;
 
   private PersonService personService;
 
```

Comment:
It's a bit belt-and-braces, but I might add a test to make sure the personService actually calls the repository.deleteById as per e.g. 

  @Test
  void shouldDeletePersonByIdFromRepository() {
    personService.deleteById(ID);
    verify(personRepository).deleteById(ID);
    verifyNoMoreInteractions(personRepository);
  }

Reply:
Sorry mate - I think this test should be in PersonServiceTest, is that alright?